cfme/storage/object_store_object.py

```diff
@@ -115,9 +116,18 @@ class ObjectStoreObjectCollection(BaseCollection):
     def all(self):
         """returning all Object Store Objects"""
         view = navigate_to(self, 'All')
-        view.toolbar.view_selector.select("Grid View")
-        objects = [self.instantiate(key=item)
-                   for item in view.entities.entity_names]
+
+        objects = []
+
+        # ToDo: use all_entity_names method as JS API issue (#2898) resolve.
+        if self.filters.has_key('provider'):                                # noqa
```

Comment:
this might be GitHub weird formatting, but only two spaces before in-line comments please.

Formatting aside, why is this line noqa'd? doesn't look like it needs it?

```python
if some.statement.true:  # noqa

Reply:
@mshriver added more pythonic way: `.has_key()` support removed in py3 so best to go with in method.